From 53387b225385acf4f629b1b70e1e9807d47742fe Mon Sep 17 00:00:00 2001
From: lunatickochiya <125438787@qq.com>
Date: Sun, 16 Feb 2025 15:10:49 +0800
Subject: [PATCH] Drop depends

---
 include/target.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/target.mk b/include/target.mk
index c36ad960fb..f1cb80a1c4 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -60,9 +60,7 @@ DEFAULT_PACKAGES.router:=\
 	ppp-mod-pppoe
 # For easy usage
 DEFAULT_PACKAGES.tweak:=\
-	autocore \
 	block-mount \
-	default-settings-chn \
 	kmod-nf-nathelper \
 	kmod-nf-nathelper-extra \
 	luci-light \
@@ -404,4 +402,4 @@ ifeq ($(TARGET_BUILD),1)
   BuildTarget?=$(BuildKernel)
 endif
 
-endif #__target_inc
\ No newline at end of file
+endif #__target_inc
-- 
2.34.1

