From c06b88f186e56bbfffb70e2c61493e6d4b7eef3a Mon Sep 17 00:00:00 2001
From: lunatickochiya <125438787@qq.com>
Date: Wed, 21 May 2025 19:13:51 +0800
Subject: [PATCH] refresh config 6.12

Signed-off-by: lunatickochiya <125438787@qq.com>
---
 target/linux/generic/config-6.12 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/generic/config-6.12 b/target/linux/generic/config-6.12
index 7a3b597e55..167b4ec5db 100644
--- a/target/linux/generic/config-6.12
+++ b/target/linux/generic/config-6.12
@@ -336,6 +336,7 @@ CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MIN=8
 # CONFIG_ARM64_AMU_EXTN is not set
 # CONFIG_ARM64_BTI is not set
 CONFIG_ARM64_CNP=y
+# CONFIG_ARM64_CONTPTE is not set
 # CONFIG_ARM64_E0PD is not set
 # CONFIG_ARM64_EPAN is not set
 # CONFIG_ARM64_ERRATUM_1024718 is not set
-- 
2.43.0

