diff --git a/modules/luci-base/po/zh_Hans/base.po b/modules/luci-base/po/zh_Hans/base.po
index 4930a9f42b..85150ab854 100644
--- a/modules/luci-base/po/zh_Hans/base.po
+++ b/modules/luci-base/po/zh_Hans/base.po
@@ -8734,7 +8734,7 @@ msgid "« Back"
 msgstr "« 后退"
 
 msgid "Source Code"
-msgstr "源代码(免费使用，禁止商用)"
+msgstr "源代码"
 
 #~ msgid "Enable promiscious mode"
 #~ msgstr "启用混杂模式"
diff --git a/modules/luci-base/root/usr/libexec/rpcd/luci b/modules/luci-base/root/usr/libexec/rpcd/luci
index 4e9340933a..72877253b4 100755
--- a/modules/luci-base/root/usr/libexec/rpcd/luci
+++ b/modules/luci-base/root/usr/libexec/rpcd/luci
@@ -640,9 +640,13 @@ local methods = {
 
 	getCPUUsage = {
 		call = function()
+			if not fs.access("/sbin/usage") then
+				return {}
+			end
+
 			local sys = require "luci.sys"
-			local cpuusage = sys.exec("top -n1 | awk '/^CPU/ {printf(\"%d%%\", 100 - $8)}'") or "?"
-			return { cpuusage = cpuusage }
+			local cpuusage = sys.exec("/sbin/usage")
+      return { cpuusage = cpuusage }
 		end
 	},
 
