diff --git a/feeds.conf.default b/feeds.conf.default
index 3702836061..26e833beba 100644
--- a/feeds.conf.default
+++ b/feeds.conf.default
@@ -1,4 +1,4 @@
 src-git packages https://github.com/lunatickochiya/packages-lunatic.git;lede-17.01
-src-git luci https://github.com/lunatickochiya/luci-lunatic.git;lede-17.01
+src-git luci https://github.com/lunatickochiya/luci-lunatic.git;17.01
 src-git routing https://github.com/lunatickochiya/routing-lunatic.git;lede-17.01
 src-git telephony https://github.com/lunatickochiya/telephony-lunatic.git;lede-17.01
