From 6c58588b2c2efb2ca4ae5fa4e5f1ce935ee14e7c Mon Sep 17 00:00:00 2001
From: Chen Minqiang <ptpt52@gmail.com>
Date: Sun, 6 Dec 2020 18:37:15 +0800
Subject: [PATCH 1/2] ath79: append okli kernel support

---
 target/linux/ath79/image/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/target/linux/ath79/image/Makefile b/target/linux/ath79/image/Makefile
index 8328be7667..99575c1a28 100644
--- a/target/linux/ath79/image/Makefile
+++ b/target/linux/ath79/image/Makefile
@@ -61,6 +61,10 @@ define Build/relocate-kernel
 endef
 
 
+define Build/append-okli-kernel
+	dd if="$(KDIR)/loader-$(word 1,$(1)).uImage" >> "$@"
+endef
+
 define Device/Default
   DEVICE_DTS_DIR := ../dts
   DEVICE_DTS = $$(SOC)_$(1)
-- 
2.34.1

