From e4af718a242b4a9822378c984eb1bdd148ef1433 Mon Sep 17 00:00:00 2001
From: Chen Minqiang <ptpt52@gmail.com>
Date: Sun, 6 Dec 2020 18:37:15 +0800
Subject: [PATCH] ath79: append/prepad okli kernel support

---
 target/linux/ath79/image/Makefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/target/linux/ath79/image/Makefile b/target/linux/ath79/image/Makefile
index 8328be7667..a90697f139 100644
--- a/target/linux/ath79/image/Makefile
+++ b/target/linux/ath79/image/Makefile
@@ -60,6 +60,16 @@ define Build/relocate-kernel
 	rm -rf $@.relocate
 endef
 
+define Build/append-okli-kernel
+	dd if="$(KDIR)/loader-$(word 1,$(1)).uImage" >> "$@"
+endef
+
+define Build/prepad-okli-kernel
+  -[ -f "$@" ] && \
+  dd if="$(KDIR)/loader-$(word 1,$(1)).uImage" of="$@".tmp bs=64k conv=sync && \
+  cat "$@" >>"$@".tmp && \
+  mv "$@".tmp "$@"
+endef
 
 define Device/Default
   DEVICE_DTS_DIR := ../dts
-- 
2.34.1

