diff --git a/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js
index a68e64b..1d933af 100644
--- a/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js
+++ b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js
@@ -79,6 +79,21 @@ return baseclass.extend({
 			);
 		}
 
+		var projectlink = document.createElement('a');
+		projectlink.append('Project OpenWrt');
+		projectlink.href = 'https://github.com/lunatickochiya';
+		projectlink.target = '_blank';
+
+		var corelink = document.createElement('a');
+		corelink.append('Powered By 2U4U');
+		corelink.href = 'https://github.com/lunatickochiya/Matrix-Action-Openwrt';
+		corelink.target = '_blank';
+
+		var sourcelink = document.createElement('placeholder');
+		sourcelink.append(projectlink);
+		sourcelink.append(' / ');
+		sourcelink.append(corelink);
+
 		var fields = [
 			_('Hostname'),         boardinfo.hostname,
 			_('Model'),            boardinfo.model + cpubench.cpubench,
@@ -94,6 +109,7 @@ return baseclass.extend({
 				systeminfo.load[2] / 65535.0
 			) : null,
 			_('CPU usage (%)'),    cpuusage.cpuusage
+			_('Source Code'),      sourcelink
 		];
 
 		if (tempinfo.tempinfo) {
