From 42be312673b937f32f480915f32871e9209a268c Mon Sep 17 00:00:00 2001
From: lunatickochiya <125438787@qq.com>
Date: Sun, 14 Apr 2024 09:29:27 +0800
Subject: [PATCH] mediatek:  CMCC RAX3000M eMMC custom U-Boot layout default

---
 target/linux/mediatek/image/filogic.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mediatek/image/filogic.mk b/target/linux/mediatek/image/filogic.mk
index f861885abf..062964ae52 100644
--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -288,6 +288,7 @@ define Device/cmcc_rax3000m-emmc-ubootmod
   DEVICE_DTS := mt7981b-cmcc-rax3000m-emmc-ubootmod
   DEVICE_DTS_DIR := ../dts
   DEVICE_PACKAGES := kmod-mt7981-firmware mt7981-wo-firmware kmod-usb3 \
+	resize2fs tune2fs losetup blkid lsblk fdisk cfdisk parted mount-utils kmod-loop autoset-resize-rootfs-script \
 	e2fsprogs f2fsck mkf2fs
   KERNEL := kernel-bin | lzma | fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
   KERNEL_INITRAMFS := kernel-bin | lzma | \
-- 
2.34.1

