From a267edc735a602391fbde8219204b4f39806b686 Mon Sep 17 00:00:00 2001
From: lunatickochiya <125438787@qq.com>
Date: Mon, 17 Mar 2025 20:54:16 +0800
Subject: [PATCH] target:add dnsmasq-full default

---
 include/target.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/target.mk b/include/target.mk
index 7526224972..9ecc3ddb06 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -43,7 +43,7 @@ DEFAULT_PACKAGES.nas:=\
 	mdadm
 # For router targets
 DEFAULT_PACKAGES.router:=\
-	dnsmasq \
+	dnsmasq-full \
 	firewall \
 	ip6tables \
 	iptables \
-- 
2.34.1

