From c1bcc2b84ca283e065149cbfd7c0e5d7b50a6197 Mon Sep 17 00:00:00 2001
From: lunatickochiya <125438787@qq.com>
Date: Thu, 6 Feb 2025 16:14:38 +0800
Subject: [PATCH 08/23] more ignore

---
 .gitignore | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.gitignore b/.gitignore
index ad04755919..6a8df8152f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -35,3 +35,7 @@ git-src
 .cproject
 .ccache
 .vscode*
+package/kochiya/*
+/feeds.conf.default
+/compile.log
+.config-*
-- 
2.34.1

