From 3ffc73cd23fead52298168ad5ef7eb491a1adfea Mon Sep 17 00:00:00 2001
From: Konstantin Demin <rockdrilla@gmail.com>
Date: Sat, 3 Dec 2022 23:32:18 +0300
Subject: [PATCH 18/23] build: use "-O2" instead of "-Os"

---
 include/target.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/target.mk b/include/target.mk
index 5330e29754..e4bba1c0d8 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -223,7 +223,7 @@ LINUX_RECONF_DIFF = $(SCRIPT_DIR)/kconfig.pl - '>' $(call __linux_confcmd,$(filt
 ifeq ($(DUMP),1)
   BuildTarget=$(BuildTargets/DumpCurrent)
 
-  CPU_CFLAGS = -Os -pipe
+  CPU_CFLAGS = -O2 -pipe
   ifneq ($(findstring mips,$(ARCH)),)
     ifneq ($(findstring mips64,$(ARCH)),)
       CPU_TYPE ?= mips64
-- 
2.34.1

